foundations/javascript_basics/arrays_and_loops.md

```diff
@@ -34,20 +34,27 @@ Test Driven Development \(TDD\) is a phrase you often hear in the dev world. It
 
 In many ways, TDD is much more productive than writing code without tests. If we didn't have the test for the adding function above, we would have to run the code ourselves over and over, plugging in different numbers until we were sure that it was working... not a big deal for a basic `add(2, 2)`, but imagine having to do that for more complicated functions, like checking whether or not someone has won a game of tic tac toe: \(`game_win(["o", null,"x",null,"x",null,"x", "o", "o"])`). If you didn't do TDD, then you might actually have to play multiple games against yourself just to test if the function was working correctly!
 
-We will teach you the art of actually writing these tests later in the course. The following practice has the tests already written out for you. All you have to do is set up the testing environment, read the specs, and write the code that makes them pass!
+We will teach you the art of actually writing these tests later in the course. For now, you will continue to work on the JavaScript exercises from before.
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 
-1. Go and review the [README of our `javascript-exercises` repository](https://github.com/TheOdinProject/javascript-exercises#readme) to set up your local environment. Once you have cloned the repository and installed Jest, review each README file prior to completing the following exercises in order:
-    - `01_helloWorld` (This exercise is intentionally very beginner friendly to ensure that you have set up everything properly!)
-    - `02_repeatString`
-    - `03_reverseString`
-    - `04_removeFromArray`
-    - `05_sumAll`
-    - `06_leapYears`
-    - `07_tempConversion`
+1. If you already completed our [Installing Node.js lesson](https://www.theodinproject.com/lessons/foundations-installing-node-js), you may move on to the next step!
```

Comment:
Nit: With the node installation lesson coming before this now, learners should have already done this. I think we can safely drop this branch of instructions and simplify this list.